server/mock/datastore_mock.go

```diff
@@ -4012,11 +4012,11 @@ func (s *DataStore) WSTEPAssociateCertHash(ctx context.Context, deviceUUID strin
 	return s.WSTEPAssociateCertHashFunc(ctx, deviceUUID, hash)
 }
 
-func (s *DataStore) MDMWindowsGetEnrolledDevice(ctx context.Context, mdmDeviceHWID string) (*fleet.MDMWindowsEnrolledDevice, error) {
+func (s *DataStore) MDMWindowsGetEnrolledDevice(ctx context.Context, mdmDeviceID string) (*fleet.MDMWindowsEnrolledDevice, error) {
```

Comment:
Unrelated to my PR, looks like the mock got merged without proper regeneration recently.

Reply:
I think `mdmDeviceHWID` was used as the name here because the HWID of the device is used to gather the table entry, see `mdm_hardware_id`  [here](https://github.com/fleetdm/fleet/blob/b43188a67190d7ea24087e32f73b9c26449e345c/server/datastore/mysql/microsoft_mdm.go#L30)

Reply:
@marcosd4h Gotcha, but this is the auto-generated mock, I believe it takes the name from whatever the argument names are in the interface, so it's normal that it re-generated it with this if the interface names were those names (that is, it's not something I changed).

Reply:
ahh I see, so we need to change the interface names then. Thanks for explaining it! I can approve the PR and tackle that later then